README.md

```diff
@@ -40,16 +40,12 @@ import (
 )
 
 func main() {
-	config := minio.Config{
-		Endpoint:        "https://s3.amazonaws.com",
-		AccessKeyID:     "YOUR-ACCESS-KEY-HERE",
-		SecretAccessKey: "YOUR-PASSWORD-HERE",
-	}
-
-	// Default is Signature Version 4. To enable Signature Version 2 do the following.
-	// config.Signature = minio.SignatureV2
+	// Requests are always secure by default. set inSecure=true to enable insecure access.
+	// inSecure boolean is the last argument for New().
 
-	s3Client, err := minio.New(config)
+	// New provides a client object backend by automatically detected signature type based
+	// on the provider.
```

Comment:
// New returns an Amazon S3 compatible client object. API copatibality (v2 or v4) is automatically determined based on the Endpoint value.